packages/trader/src/Modules/Trading/Helpers/contract-type.tsx

```diff
@@ -1,6 +1,26 @@
 import React from 'react';
 import { localize } from '@deriv/translations';
 
+type Category = { value: string; text: string };
+type TContract_type = {
```

Comment:
TContractType is the convention. I think we use Pascal case for types.

Reply:
Noted, thank you